docs/platform-parts/container/container-integration.md

```diff
@@ -204,22 +204,22 @@ To add a container using Carthage:
 1) Create a Cartfile if you don't already have one, or open an existing Cartfile.  
 2) Add the following line to your Cartfile.
 
-```bash
-$ git "git@github.com:user/myweatherapp-ios-container.git" "v1.0.0"
-```
+    ```bash
+    $ git "git@github.com:user/myweatherapp-ios-container.git" "v1.0.0"
+    ```
 
 3) Create a `Cartfile.resolved` file if you don't have one or open your existing `Cartfile.resolved` file.  
 4) Add the following line to your `Cartfile.resolved` file:
 
-```bash
-$ git "git@github.com:user/myweatherapp-ios-container.git" "v1.0.0"
-```
+    ```bash
+    $ git "git@github.com:user/myweatherapp-ios-container.git" "v1.0.0"
```

Comment:
Since you're touching these lines anyway, you might as well just fix `MD014` as well (remove the dollar sign).

    commands-show-output Dollar signs used before commands without showing output